### PR TITLE
Added validation stage to the sdk build

### PR DESCRIPTION
**Pull request by reutshal**
## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,6 +32,40 @@ jobs:
...
               coveralls
+        - run:
+            name: test validate files and yaml
+            command: |
+              . .tox/py37/bin/activate
+
+              git clone https://github.com/demisto/content.git
+              cd content
+
+              if [[ $CIRCLE_BRANCH != master && $FAKE_MASTER != true ]]; then
```
Comment (yaakovi) [inline .circleci/config.yml:43]: what's `FAKE_MASTER` supposed to mean?

**Comment by reutshal** [comment]
> so,
> regular builds will run validate of a single pack
> and master builds will run validate of content's master, create id set, and create artifacts?

Exactly.

File: .circleci/config.yml (hunk 1 above)
Comment (reutshal) [reply .circleci/config.yml:43]: Like we talked today, it is a way to trigger the sdk build as it would run on master

Comment (yaakovi) [reply .circleci/config.yml:43]: in that case, you can switch the condition to depend only on that variable (when not defined, will enter the condition scope):
```suggestion
              if [[ -z "$FAKE_MASTER" ]]; then
```

**Pull request merged**